.github/workflows/release.yml

```
@@ -0,0 +1,27 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    branches:
+      - master
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

```suggestion
          fetch-depth: 0 // get the entire history
```

for future readers can we include this so people don't need to look it up?